### PR TITLE
Sparse plots

### PR DESCRIPTION
**Pull request by caryan**
Read sequence file functions now return lists of time amplitude pairs.  Plots are closer to hardware (i.e. no interpolation) and helpful for performance for plots with lots of time amplitude pairs.

### CODE REVIEW

File: QGL/drivers/APS2Pattern.py [hunk 1]
```diff
@@ -770,7 +770,7 @@ def start_new_seq():
 					seqs['mod_phase'][-1] = np.append(seqs['mod_phase'][-1], freq[nco_select]*np.arange(count) + accumulated_phase[nco_select] + phase[nco_select] + frame[nco_select])
 					accumulated_phase += count*freq
 				else:
-					phase_rad = 2*np.pi * (instr.payload & 0xffffffff) / 2**28
+					phase_rad = 2*np.pi * np.int32(instr.payload & 0xffffffff) / 2**28
```
Comment (dieris) [inline QGL/drivers/APS2Pattern.py]: This overflows: 
> Python int too large to convert to C long

Does ` np.int32((instr.payload & 0xffffffff) / 2**28)` achieve the same goal?

Comment (caryan) [inline QGL/drivers/APS2Pattern.py]: Can you add the h5 file that does this?  I don't see how it's possible after the `& 0xffff_ffff`.  I want to convert and then divide to preserve the sign bit.

Comment (caryan) [inline QGL/drivers/APS2Pattern.py]: Although it's probably easier to just treat everything as unsigned and wrap negative phases.

Comment (dieris) [inline QGL/drivers/APS2Pattern.py]: [Rabi-APSII7.zip](https://github.com/BBN-Q/QGL/files/247706/Rabi-APSII7.zip)

**Pull request merged**